src/integration/pages/homePage/canonicalTests.js

```diff
@@ -0,0 +1,11 @@
+import { runCoreCanonicalTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+import pageContentTests from './pageContentTests';
+import messageBannerTest from './messageBannerTests';
+
+export default service => {
+  runCrossPlatformTests(service);
+  runCoreCanonicalTests();
+  pageContentTests();
+  messageBannerTest();
```

Comment:
We do now support AMP for homepage - we managed to get AMP validation working, so we can now use the AMP tests. My apologies, when we first spoke about these tests, AMP was not supported, but now it is. 

Happy to help out with this if needed, it would be good to get as many tests as we possibly can, and avoid dupes